cypress/e2e/checkout.cy.js

```diff
@@ -1,11 +1,57 @@
+import CheckoutFormPageObject from '../support/pages/checkoutForm.pageObject';
+import HomeAndCataloguePageObject
```

Comment:
In your tests I see homeAndCatalogue page object but I don't see this in your PR. Please add this file.

Reply:
@AnnaPalamarchukQA 
This file was by default in the repository, I used it but did not change it, so it is not added to the request pool